### ISSUE TITLE
[step2.2][jitData] tested individually the data are correctly perturbed in position, roation, scale but not with jitData

### ISSUE THREAD

**Issue by msprj2010** [msprj2010/_eSDC_#1]
**Code:** _Step2.2_jitData.ipynb

**Description:**
Tested individually, jitShift, jitRot and jitCrop perturbe correctly in position, in roation and in scale the images (see Screenshot.png, cell 27). But combined in jitData the images are perturbed but their colors are altered  (see Screenshot.png, cell 26). If we change the theta value (15 deg by 90 deg), it is affected only the color spectrum of the images.

**In summary:**
- jitData does not combine the 3 functions: jitShift, jitRot, jitCrop
- jitData results are images that the colors have been altered
- see Screenshot.png
.
